sys/log/full/include/log/log.h

```diff
@@ -97,6 +97,7 @@ typedef int (*lh_append_mbuf_body_func_t)(struct log *log,
 typedef int (*lh_walk_func_t)(struct log *,
         log_walk_func_t walk_func, struct log_offset *log_offset);
 typedef int (*lh_flush_func_t)(struct log *);
+typedef uint16_t (*lh_read_entry_len_func_t)(struct log *, const void *dptr);
```

Comment:
It's consistent with the other prototypes. That's the reason I made it that way. For function pointers it does not matter what the name of the variable is anyways.

Reply:
So you are saying that is consistent with inconsistency?

Reply:
Yes, it is, do you want me to fix it ? Just say so. This is a PR to add support for a targeted feature. In the past we would open separate PRs for refactoring Vs changes made for a feature. I can change it in a separate PR later. cc: @sjanc

Reply:
I leave it up to you. If you do, I will click `Resolve conversation` button so others can focus on other things. I looked at the code and tested this change and I don't see any improvement. Combability is not preserved and with the way TLV is done it will not be better in the future if more changes are needed. I can't approve this. Even if you fix all small things I pointed out I can't see any reason to add this new field let alone the way it is added. If you find someone else willing to approve so be it.

Reply:
Are you sure compatibility is not preserved ? Maybe it does not work on hardware you are using. Is it possible for it to try on the same hardware as mine ?

Reply:
Anyways, let's focus on this comment. You can re-review once I am done making changes.